src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -139,9 +144,46 @@ public bool IsOneRowContent
             }
         }
 
+
+        /// <summary>
+        /// Number of items in the overall list if node output is a list
+        /// </summary>
+        public int NumberOfItemsWT
+        { 
+            get { return numberOfItems; }
+            set
+            {
+                numberOfItems = value;
+                IsCollection = true;
+                RaisePropertyChanged("NumberOfItemsWT");
+                
+            }
+        }
+
+        /// <summary>
+        /// Indicates if number of list items is shown
+        /// </summary>
+        public bool ShowNumberOfItems
+        {
+            get { return IsCollection; }
+        }
+
+        /// <summary>
+        /// Indicates if the items are lists
+        /// </summary>
+        public bool IsCollection
+        {
+            get { return _isCollection; }
+            set {
+                _isCollection = value;
+                RaisePropertyChanged("ShowNumberOfItems");
```

Comment:
As `ShowNumberOfItem` returns same value as `IsCollection`, why not remove `ShowNumberOfItems` and use `IsCollection`?